java/src/jmri/jmrit/logixng/actions/ActionListenOnBeans.java

```diff
@@ -228,7 +228,22 @@ public String getLongDescription(Locale locale) {
     /** {@inheritDoc} */
     @Override
     public void setup() {
-        // Do nothing
+        for (NamedBeanReference ref : getReferences()) {
+            if (ref.getType().name().equals("EntryExit")) {
```

Comment:
`ref.getType()` returns the enum `NamedBeanType` so I suggest that you change this to:

`if (ref.getType() == NamedBeanType.EntryExit) {`

This also applies to ActionListenOnBeansXml.